modules/cas_cache/utils/utils_mpool.c

```diff
@@ -98,20 +124,24 @@ void *cas_mpool_new_f(struct cas_mpool *mpool, uint32_t count, int flags)
 	return items;
 }
 
-void *cas_mpool_new(struct cas_mpool *mpool, uint32_t count)
+void *env_mpool_new(struct env_mpool *mpool, uint32_t count)
 {
-	return cas_mpool_new_f(mpool, count, mpool->flags);
+	return env_mpool_new_f(mpool, count, mpool->flags);
 }
 
-void cas_mpool_del(struct cas_mpool *mpool,
+bool env_mpool_del(struct env_mpool *mpool,
 		void *items, uint32_t count)
 {
 	env_allocator *allocator;
 
-	allocator = cas_mpool_get_allocator(mpool, count);
+	allocator = env_mpool_get_allocator(mpool, count);
```

Comment:
early return in case items == NULL might be beneficial to avoid hitting the problematic case in line 144.